services/common/src/interfaces/permits/permit.interface.ts

```diff
@@ -23,4 +23,5 @@ export interface IPermit {
   site_properties: IMineType;
   permit_prefix: string;
   mine_guid?: string;
+  amendments?: Partial<IPermitAmendment>[];
```

Comment:
Wait, we have `amendments` and `permit_amendments`.

Reply:
Oh whoops I forgot to remove this after some testing. Thanks for catching this.